packages/api-key-stamper/src/tink/elliptic_curves.ts

```diff
@@ -7,6 +7,10 @@
  */
 
 import * as Bytes from "./bytes";
+import {
+  DEFAULT_JWK_MEMBER_BYTE_LENGTH,
+  uint8ArrayFromHexString,
+} from "@turnkey/encoding";
```

Comment:
Mmm, this seems a bit off because we have a vendor'd library import our encoding lib :/